src/Config/Config.php

```diff
@@ -75,14 +78,13 @@ public function getMigrationPaths(): array
      * @inheritDoc
      * @throws \UnexpectedValueException
      */
-    public function getSeedPaths(): array
+    public function getSeedPath(): string
     {
         if (!isset($this->values['paths']['seeds'])) {
             throw new UnexpectedValueException('Seeds path missing from config file');
         }
-
-        if (is_string($this->values['paths']['seeds'])) {
-            $this->values['paths']['seeds'] = [$this->values['paths']['seeds']];
+        if (is_array($this->values['paths']['seeds']) && isset($this->values['paths']['seeds'][0])) {
```

Comment:
What is the point of this being an array if it's hard-coded to index 0?

Reply:
I was trying to keep compatibility with all the tests that provide a list of paths.